cmd/state-installer/installer.go

```diff
@@ -37,7 +37,7 @@ func main() {
 		if panics.HandlePanics(recover(), debug.Stack()) {
 			exitCode = 1
 		}
-		if err := events.WaitForEvents(1*time.Second, rollbar.Close, authentication.LegacyClose); err != nil {
+		if err := events.WaitForEvents(1*time.Second, analytics.Wait, rollbar.Close, authentication.LegacyClose); err != nil {
```

Comment:
I am fairly sure that adding `analytics.Wait` here, fixes the root problem ie., the bug observed by Jeremy.

As the analytics functions may trigger a logging error event that creates a rollbar message, we should wait for the analytics (background) processes to finish before we are closing the rollbar.  That will avoid this issue in the future.